src/odemis/gui/cont/microscope.py

```diff
@@ -94,13 +94,14 @@ def Enable(self, enabled=True):
 class ChamberButtonController(HardwareButtonController):
     """ Controller that allows for the more complex state updates required by the chamber button """
 
-    def __init__(self, btn_ctrl, va, main_data, show_pressure=True):
+    def __init__(self, btn_ctrl, va, main_data, pressure_ctrl=None):
         """
 
         :param btn_ctrl: (ImageTextToggleButton) Button that controls and displays the chamber state
         :param va: (VigillantAttribute) The chamber state
         :param main_data: (MainGUIData) GUI microscope model
-        :param show_pressure (bool): whether to show the numerical pressure value on the button
+        :param pressure_ctrl (wxStaticText, None): control for showing the numerical pressure value.
+            If None, show the pressure on the button.
```

Comment:
Shouldn't it be shown at all, when it is None? For the SecomStateController you had before that the pressure is simply not shown at all if I understood correctly. And now you will show the pressure on top of the button, which is not the standard behavior (just pump/vent labels).

We have now defined two behaviors:
1. The pressure is shown as text below the pump/vent button.
2. No pressure is shown, just the pump/vent button as usual.

Reply:
The behaviour before (for secom I guess?) was to show the pressure on top of the button, so I didn't want to change that and only added functionality for the FastEM without modifying the behaviour in other systems.

Reply:
Ok, I didn't know this. In your first version you had show_pressure = False, that is why I thought it should not be shown for the SecomStateController.

Reply:
I am now wondering how it works in the SPARC. There we also have a chamber...